src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/helper/EnvelopeCreator.java

```diff
@@ -123,7 +123,7 @@ public static Envelope envelopeNotified() {
             ImmutableMap.of("name1", "value1"),
             "1111002.pdf",
             "test",
-            "Other"
+            "other"
```

Comment:
can an existing enum be used here?

Reply:
There's no existing enum. I'm introducing constants in another PR, introducing document subtypes. Will consider enums there.